Appraisals

```diff
@@ -130,6 +130,7 @@ if RUBY_VERSION >= '2.2.2' && RUBY_PLATFORM != 'java'
     gem 'sinatra'
     gem 'sqlite3'
     gem 'activerecord', '< 5.1.5'
+    gem 'database_cleaner'
```

Comment:
Why do we need `database_cleaner`? I suspect our tests should probably never be dependent upon the state of a database.

Reply:
I've added instantiation tests as well as sql tests for `ActiveRecord#create` and `#create` creates a db record. So to be able to perform the test multiple times with different configurations. DatabaseCleaner was needed, it works nicely by wrapping test in a transaction then reverting just that one transaction. The end result is that the database we use for tests is kept clean after every test run.